### DISCUSSION TITLE
A geophysical tour of mid-ocean ridges

### DISCUSSION THREAD

**Discussion by giscus** [leouieda/website#250] [Comments]
# A geophysical tour of mid-ocean ridges

This tutorial was developed for Transform 2022, the Software Underground’s virtual festival of open Geoscience. Watch the recording on YouTube Run the code online: launch Binder Original source...

https://www.leouieda.com/blog/midocean-ridges.html

**Comment by leouieda** [comment]
Just testing [giscus](https://github.com/giscus/giscus) for comments.

**Reply by leouieda** [reply]
It works!
